redux/selectors/filters.js

```diff
@@ -0,0 +1,12 @@
+import { createSelector } from 'reselect-change-memoize'
+import setQueryParameters from 'common/utilities/setQueryParameters'
+
+export default createSelector(
+  'filters',
+  state => state.filters,
+  filtersState => {
+    setQueryParameters({ showFailed: filtersState.showFailed, projectQuery: filtersState.projectQuery })
```

Comment:
redux stuff is kind of agnostic to the website bit. We can use it with React Native for example. 

I would move this to the React component's `componentDidReceiveProps()` because it makes sense that the react component is responsibility for doing websitey things. 

This is to do with each part of the code having their own responsibilities and not bleeding them over. 

https://en.wikipedia.org/wiki/Single_responsibility_principle
or maybe https://en.wikipedia.org/wiki/Separation_of_concerns